<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="p1bar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="p2bar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>120sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>5 min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
